socialhome/notifications/templates/notifications/base.html

```diff
@@ -0,0 +1,25 @@
+{% load i18n %}<!doctype html>
```

Comment:
Missing a newline here, isn't it?

Reply:
This is actually quite common in the Django world. I guess(?) it is because the HTML spec says a HTML document should start with the doctype. I have no idea if it causes problems or not but having the doctype on a separate line would include empty lines before the doctype. I think it's safer to leave it as is, though I couldn't find a clear answer to whether it could cause problems. Many examples and Django admin templates themselves do it like this.